content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -1,1440 +1,1363 @@
+
+### Features Overview
+
+Portenta X8 integrates two main programming experiences: the **Yocto Linux** and popular **Arduino** environments.
+
+![Portenta X8 Features Overview](assets/portenta-x8-functionality-overview.png "Portenta X8 Features Overview")
+
+To explore specific sections in more detail, please click on the links below that interest you:
+
+* [Linux Environment](#linux-environment)
+* [Arduino Environment](#arduino-environment)
+* [Arduino Cloud](#portenta-x8-with-arduino-cloud)
+
+### Portenta X8: Linux & Arduino Interoperability
+
+The Portenta X8 runs a **Yocto** based Linux distribution. It allows users to upload sketches to the M4 core of the STM32H7 using the Arduino IDE. The process of uploading sketches may seem familiar to Arduino users, but on Portenta X8, the system uses a service called `monitor-m4-elf-file.service`.
+
+This service monitors a specific directory for updated sketches and flashes the M4 core with the new code via OpenOCD. This feature enables seamless integration between the Linux and Arduino environments on the board.
+
+The Portenta X8 employs **Remote Procedure Call (RPC)** mechanisms to enable communication between the Linux system and the Arduino cores. RPC allows the two processors (M4 and M7 cores) to exchange data, with the M7 core handling data transfers between the M4 core running Arduino sketches and the Linux environment.
+
+The communication uses **MessagePack-RPC**, a lightweight protocol for serialized data transfer, supporting various communication protocols like OpenAMP, SPI, and TCP/IP. This setup ensures efficient interaction between the Linux side and Arduino sketches without the complexity of traditional dual-core processing.
+
+The Portenta X8 offers a powerful, versatile platform by combining Linux capabilities with Arduino’s ease of use. Users can deploy and manage device-independent applications using Docker containers while also taking advantage of Arduino’s rich ecosystem for embedded development. This hybrid environment supports flexible development, making it ideal for complex, multifunctional projects.
+
+***For more in-depth details on fundamentals of the Portenta X8, please refer to this [documentation](https://docs.arduino.cc/tutorials/portenta-x8/x8-fundamentals/).***
+
+## Portenta X8 OS Image Update
+
+It is recommended that you check every now and then to see if your Portenta X8 image version is up to date to have the latest security updates.
+
+***To review the image version history, refer to the [__Portenta X8 firmware release notes__](https://docs.arduino.cc/tutorials/portenta-x8/x8-firmware-release-notes/).***
+
+There are four ways to update your Portenta X8 are described:
+
+* [Update for OS release V.399](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing/#update-for-os-release-v399)
+* [Update through Out-of-the-box experience](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing/#update-through-out-of-the-box-experience)
```

Comment:
Could we please rename everywhere Out-of-the-box with Arduino Linux Wizard?

Reply:
Updated!